pkg/apis/core/types_shoot.go

```diff
@@ -401,7 +401,8 @@ type HibernationSchedule struct {
 
 // Kubernetes contains the version and configuration variables for the Shoot control plane.
 type Kubernetes struct {
-	// AllowPrivilegedContainers indicates whether privileged containers are allowed in the Shoot (default: true).
+	// AllowPrivilegedContainers indicates whether privileged containers are allowed in the Shoot.
+	// Defaults to true for kubernetes versions below v1.25.
```

Comment:
Nit
Similar changes for other occurrences.
```suggestion
	// AllowPrivilegedContainers indicates whether privileged containers are allowed in the Shoot.
	// Defaults to true for Kubernetes versions below v1.25.
```